cmd/helpMessages.go

```diff
@@ -76,11 +76,16 @@ Download a single file with a SAS using piping (block blobs only):
 Download an entire directory with a SAS:
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/directory]?[SAS]" "/path/to/dir" --recursive=true
 
+A note about wildcards in URLs:
+
+The only usage of a wildcard in a URL that is supported is the final, trailing /*, and in the container name when not specifying a blob name.
+If you happen to use the character * in the name of a blob, please manually encode it to %2A to avoid it being treated as a wildcard character.
+
 Download a set of files with a SAS using wildcards:
```

Comment:
I think we should remove these two "with wildcards" examples, since they are not accomplishing the original purpose (of showing how to use wildcards in URLs) because that original purprse is no longe supported.

Reply:
Maybe we could change the previous example instead. I can't describe it in a comment (too difficult), so I'll push a small change  to the PR ;-)

Reply:
I think the best way we could phrase it is
"To put the contents of a folder directly into the destination rather than under a sub-directory, append a trailing /* to the URL."

Reply:
Oh wow, that's perfect!  Way better than what I had in mind, so I won't push anything.